src/impl/EspNowNode.cpp

```diff
@@ -131,6 +135,12 @@ bool EspNowNode::setup() {
   bool presumably_valid_host_mac_address = _preferences.espNowGetMacForHost(_esp_now_host_address);
   if (presumably_valid_host_mac_address) {
     log("Presumably valid MAC address loaded.", ESP_LOG_INFO);
+    uint8_t channel = 1;
+    _preferences.espNowGetChannelForHost(&channel);
+    log("loaded channel " + std::to_string(channel), ESP_LOG_INFO);
+    ESP_ERROR_CHECK(esp_wifi_set_promiscuous(true));
```

Comment:
Suggest moving these three repeated lines (used here and L:180) into a class member function (set_promiscuous, set_channel, set_promiscuous)

Reply:
I tested changing the channel without enabling/disabling promiscuous mode, and it worked.  That enable/disable was something that was necessary in the past, although that may have been when the device is also connected to a WiFi network.  But either way, testing without enable/disable was fine, so I've removed those lines, and that removes the need for the separate function.